src/libs/Ideogram/openapi.yaml

```
@@ -1838,18 +1838,18 @@
           description: The number of images to generate.
           default: 1
         resolution:
-          $ref: '#/components/schemas/Resolution'
+          $ref: '#/components/schemas/ResolutionV3'
         color_palette:
           $ref: '#/components/schemas/ColorPaletteWithPresetNameOrMembers'
       example:
-        aspect_ratio: ASPECT_10_16
-        style_type: REALISTIC
+        aspect_ratio: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)